### PR TITLE
Polished The UI

### PR DESCRIPTION
**Pull request by Adyel**
Added Layout, LLVM coding style, and some unknown bugs. :smiley_cat:

### CODE REVIEW

File: src/frontend/AboutFile.ui [hunk 1]
```diff
@@ -1,160 +1,156 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (mominul) [inline src/frontend/AboutFile.ui]: about_layout
There is too much space on the right side.

**Comment by mominul** [comment]
Great Work! :tada:

File: src/frontend/AboutFile.ui (hunk 1 above)
Comment (Adyel) [reply src/frontend/AboutFile.ui]: Ok, I'll fix that.

**Comment by Adyel** [comment]
Fixing the conflict from my fork and recreating PR.

**Pull request closed without merging**